src/gui/guiFormSpecMenu.h

```diff
@@ -259,7 +259,7 @@ class GUIFormSpecMenu : public GUIModalMenu
 	void acceptInput(FormspecQuitMode quitmode=quit_mode_no);
 	bool preprocessEvent(const SEvent& event);
 	bool OnEvent(const SEvent& event);
-	bool doPause;
+	bool doPause = false;
```

Comment:
Makes the additional line in `guiFormSpecMenu.cpp`/`GUIFormSpecMenu::create` superfluous